informer/src/handler/subscription.rs

```diff
@@ -14,28 +14,38 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-use crate::{ColdEvents, EventTags, Events, Params, Sink};
+use crate::{ColdEvents, EventTags, Events, Params, Sink, SubscriptionId};
 use jsonrpc_core::futures::Future;
 
 #[derive(Clone)]
 enum ConnectionState {
     Connected,
 }
 
+#[derive(Clone)]
+pub enum ColdState {
+    Has,
+    NotApplicable,
+    Removed,
```

Comment:
How about using two enum variants "Subscribed", and "UnSubscribed"?
I don't understand the purpose of the "NotApplicable". Could you explain it to me?

Reply:
I explained in the previous comment.